src/assets/styles/colors.scss

```diff
@@ -1 +1,6 @@
-// define app colors
+$primary: #3F51B5;
+$white: #ffffff;
```

Comment:
All colors are following uppercase convention but this one is not. Maybe you can unify this.
Also like I said in other comment I'm not sure if those changes are even needed currently.